tests/testsuite/tree.rs

```diff
@@ -1888,15 +1888,12 @@ fn depth_workspace() {
         .file("c/src/lib.rs", "")
         .build();
 
-    p.cargo("tree")
+    p.cargo("tree --depth workspace")
```

Comment:
This should be squashed into the previous commit.

Every commit should be atomic which includes making sure the tests pass.

Reply:
Squashing would be better and I'm gonna do so, but I think that the test may pass as it is, because the output is not updated yet